### PR TITLE
Try enabling more Python 3 testing.

### PR DESCRIPTION
**Pull request by jmchilton**
Lets see if these pass on Travis.

### CODE REVIEW

**Comment by jmchilton** [comment]
Well that found a bunch of Python 3 bugs at least...

**Comment by nsoranzo** [comment]
Down to 2 failures! I have another commit ready that will fix the first once https://github.com/galaxyproject/galaxy-lib/pull/89 is merged.

**Comment by nsoranzo** [comment]
Rebased and green!

File: planemo/tool_builder.py [hunk 1]
```diff
@@ -300,7 +300,7 @@ def _build_galaxy(**kwds):
     # process raw cite urls
     cite_urls = kwds.get("cite_url", [])
     del kwds["cite_url"]
-    citations = map(UrlCitation, cite_urls)
+    citations = list(map(UrlCitation, cite_urls))
```
Comment (peterjc) [inline planemo/tool_builder.py]: How about using a list comprehension here?

```python
citations = [UrlCitation(url) for url in cite_urls]
```

Or, using the ``_`` variable:

```python
citations = [UrlCitation(_) for _ in cite_urls]
```

Comment (nsoranzo) [reply planemo/tool_builder.py]: Funnily we discussed this yesterday in a `galaxy-lib` PR: https://github.com/galaxyproject/galaxy-lib/pull/88#discussion_r169907266

**Comment by jmchilton** [comment]
Wonderful, wonderful - thanks so much @nsoranzo - this is amazing.

**Pull request merged**